python/dask_cudf/dask_cudf/backends.py

```diff
@@ -426,3 +428,70 @@ def sizeof_cudf_dataframe(df):
 @_dask_cudf_nvtx_annotate
 def sizeof_cudf_series_index(obj):
     return obj.memory_usage()
+
+
+try:
+
+    # Define "cudf" backend engine to be registered with Dask
+    from dask.dataframe.backends import DataFrameBackendEntrypoint
+
+    class CudfBackendEntrypoint(DataFrameBackendEntrypoint):
+        @staticmethod
+        def from_dict(data, npartitions, orient="columns", **kwargs):
+            from dask_cudf import from_cudf
+
+            if orient != "columns":
+                raise ValueError(f"orient={orient} is not supported")
+            # TODO: Use cudf.from_dict
+            # (See: https://github.com/rapidsai/cudf/issues/11934)
+            return from_cudf(
+                cudf.DataFrame(data),
+                npartitions=npartitions,
+            )
+
+        @staticmethod
+        def read_parquet(*args, engine=None, **kwargs):
+            from dask_cudf.io.parquet import CudfEngine
+
+            with config.set({"dataframe.backend": "pandas"}):
+                return dd.read_parquet(
+                    *args,
+                    engine=CudfEngine,
+                    **kwargs,
+                )
+
+        @staticmethod
+        def read_json(*args, engine=None, **kwargs):
+            with config.set({"dataframe.backend": "pandas"}):
+                return dd.read_json(*args, engine=cudf.read_json, **kwargs)
+
+        @staticmethod
+        def read_orc(*args, **kwargs):
+            from dask_cudf.io import read_orc
+
+            return read_orc(*args, **kwargs)
+
+        @staticmethod
+        def read_csv(*args, **kwargs):
+            from dask_cudf.io import read_csv
+
+            chunksize = kwargs.pop("chunksize", None)
+            blocksize = kwargs.pop("blocksize", "default")
```

Comment:
`blocksize` is now not passed to the `dask_cudf` `read_csv` function. Although looking at the logic that seems safe.

Reply:
Right, I *think* what we have here is safe?